site/content/en/docs/concepts/preemption.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Preemption"
+date: 2024-05-28
+weight: 7
+description: >
+  Preemption is the process of evicting one or more admitted Workloads to accommodate another Workload.
+---
+
+In a preemption, preempted Workloads are also called preemptees, and the ClusterQueue that
+the Workload belongs to is called a target ClusterQueue.
+Converselly, the Workload being accommodated is called the preemptor, and the ClusterQueue that the
+Workload belongs to is called a preempting ClusterQueue.
+
+## Reasons for preemption
+
+A Workload can preempt one or more Workloads if it is admitted in a [ClusterQueue with preemption enabled](/docs/concepts/cluster_queue/#preemption)
+and:
```

Comment:
Do all the items in the list have to be true for the condition to be met? Or is it any?

I think that you should clarify by saying:

"(...) and any of the following events happen" OR "(...) and all the following events happen"

Reply:
It is any, thanks!